cmd/kubelet/app/options/options.go

```diff
@@ -320,8 +320,6 @@ func (f *KubeletFlags) AddFlags(mainfs *pflag.FlagSet) {
 	fs.MarkDeprecated("register-schedulable", "will be removed in a future version")
 	fs.StringVar(&f.ExperimentalMounterPath, "experimental-mounter-path", f.ExperimentalMounterPath, "[Experimental] Path of mounter binary. Leave empty to use the default mount.")
 	fs.MarkDeprecated("experimental-mounter-path", "will be removed in 1.25 or later. in favor of using CSI.")
-	fs.StringVar(&f.CloudConfigFile, "cloud-config", f.CloudConfigFile, "The path to the cloud provider configuration file. Empty string for no configuration file.")
```

Comment:
In addition to warning in the release notes, please also check for usage in-tree. hack/local-up-cluster.sh still uses it and now fails.

Reply:
=> https://github.com/kubernetes/kubernetes/pull/131841

Reply:
Sorry for that.